integrations/schemas/ort-configuration-schema.json

```diff
@@ -382,9 +382,6 @@
                 "sslrootcert": {
                     "type": "string"
                 },
-                "parallelTransactions": {
```

Comment:
Please add to the commit message that this is a fixup for f7dd71961496b92f293a97ef39bc9b9cfba68da9 which removed the use of this parameter.